File: test-automation-core/src/main/java/com/nortal/test/core/exceptions/TestExecutionException.java
package com.nortal.test.core.exceptions;

/**
 * A generic exception to be thrown when continuing scenario execution is no longer possible
 */
public class TestExecutionException extends RuntimeException {

	public TestExecutionException(final String message) {
		super(message);
	}

	public TestExecutionException(final Throwable cause) {
		super(cause);
	}

	public TestExecutionException(final String message, final Throwable cause) {
		super(message, cause);
	}
}

File: test-automation-core-old/src/main/java/com/nortal/test/core/exceptions/TestExecutionException.kt
package com.nortal.test.core.exceptions

import java.lang.RuntimeException

/**
 * A generic exception to be thrown when continuing scenario execution is no longer possible
 */
open class TestExecutionException : RuntimeException {
    constructor(message: String?) : super(message) {}
    constructor(cause: Throwable?) : super(cause) {}
    constructor(message: String?, cause: Throwable?) : super(message, cause) {}
}